apps/console/src/features/organizations/hooks/use-organization-switch.ts

```diff
@@ -79,9 +89,39 @@ const useOrganizationSwitch = (): UseOrganizationSwitchInterface => {
         return response;
     };
 
+    /**
+     * Handles the organization switch in the legacy mode.
+     * @param breadcrumbList - Breadcrumb list
+     * @param org - Organization to switch to.
+     */
+    const switchOrganizationInLegacyMode = (breadcrumbList: BreadcrumbList, org: GenericOrganization): void => {
+
+        let newOrgPath: string = "";
+
+        if (
+            breadcrumbList && breadcrumbList.length > 0
```

Comment:
Optional chaining is a new operator introduced to get rid of && chains. Shall we use that here as well?

```suggestion
            breadcrumbList?.length > 0
```